app/server/management/commands/create_admin.py

```diff
@@ -20,12 +20,13 @@ def handle(self, *args, **options):
 
         try:
             super().handle(*args, **options)
-        except IntegrityError:
+        except (IntegrityError, CommandError):
             self.stderr.write(f'User {username} already exists.')
```

Comment:
Could `(IntegrityError, CommandError)` exception mean something different user already exists? Should we analyze exception more (not only type/class)?

Reply:
Since the entire method is just used to create the superuser I do not think it can mean anything different as far as I understand the django documentation.

Reply:
No. Exception is [raised](https://github.com/django/django/blob/2.2.10/django/contrib/auth/management/commands/createsuperuser.py#L63-L169) in many cases.

Even `CommandError` is raised in many cases:
* if [`username == ''`](https://github.com/django/django/blob/2.2.10/django/contrib/auth/management/commands/createsuperuser.py#L88-L89) or [`username is None`](https://github.com/django/django/blob/2.2.10/django/contrib/auth/management/commands/createsuperuser.py#L141-L142),
* [here](https://github.com/django/django/blob/2.2.10/django/contrib/auth/management/commands/createsuperuser.py#L153-L154),
* if [this function](https://github.com/django/django/blob/2.2.10/django/contrib/auth/management/commands/createsuperuser.py#L197-L211) returned other than `None` (**three** cases),
* [on `ValidationError`](https://github.com/django/django/blob/2.2.10/django/contrib/auth/management/commands/createsuperuser.py#L162-L163).

That why I propose to `print(err)` only or similar.
However I don't understand how to understand if error was fatal.

And confirm: I also don't see `IntegrityError`.